src/OrbitClient.php

```diff
@@ -144,6 +150,11 @@ public function getOptions()
         return $this->options;
     }
 
+    public function setFlushCacheItems(bool $flushCacheItems): void
```

Comment:
This is in the middle of a load of groupped `get` methods. Should this be moved somewhere else?